packages/venia-concept/src/components/Pagination/navButton.css

```diff
@@ -1,8 +1,10 @@
-.arrow {
+.buttonArrow {
```

Comment:
It would probably have been fine to leave these names as `.arrow` and `.inactive`, but this also works.